kuma/javascript/src/__snapshots__/page.test.js.snap

```diff
@@ -775,9 +775,9 @@ Array [
           ike
            
           <time
-            dateTime="2019-01-02T03:04:05"
+            dateTime="2019-01-02T03:04:05Z"
```

Comment:
This snapshot test was failing because I was feeding it a date without a timezone, and my docker container is in a different timezone than my laptop is. By forcing the input data to UTC, the problem goes away.

Reply:
I'm stunned that we're using snapshots. I did try it briefly for a project but I found it counterproductive. It added more overheads and just a chore to mass-update them without even looking at the details. To me, they became a test of the tests rather than a test of the functionality. 

By the way, what I ended up doing was revert all attempts at using snapshots and instead embarked to rewrite all enzyme tests with react-testing-library instead.

Reply:
I'm not yet convinced by the snapshot tests. But they're basically free, and haven't been too much of a headache so far. I don't know what a good alternative is. This is an example where having the snapshot test caught a fairly subtle bug, so I'm not ready to discard them.